guides/micronaut-oauth2-linkedin/groovy/src/main/groovy/example/micronaut/LinkedInMe.groovy

```diff
@@ -0,0 +1,34 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+import io.micronaut.core.annotation.NonNull
+
+import javax.validation.constraints.NotBlank
+
+@CompileStatic
+@Introspected
+class LinkedInMe {
+
+    @NonNull
+    @NotBlank
+    final String id
+
+    @NonNull
+    @NotBlank
+    final String localizedFirstName
+
+    @NonNull
+    @NotBlank
+    final String localizedLastName
+
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.